lesson2/Goods/src/com/company/Goods.java

```diff
@@ -0,0 +1,113 @@
+package com.company;
+
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Class, which save objects
+ */
+public class Goods {
+    private ListGoods[] listgoods;
+    private String type;
+    private String name;
+    private double count;
+    private double cost;
+
+    class ListGoods {
+        private String type;
+        private String name;
+        private double count;
+        private double cost;
+        private double totalCost;
+
+        /**
+         * Inicial params
+         *
+         * @param type - type of goods
+         * @param name - name of type of goods
+         * @param count - count of goods
+         * @param cost - cost of goods
+         */
+        public ListGoods(String type, String name, double count, double cost) {
+            this.type = type;
+            this.name = name;
+            this.count = count;
+            this.cost = cost;
+        }
+
+        public String getType() {
+            return this.type;
+        }
+
+        public String getName() {
+            return this.name;
+        }
+
+        public double getCount() {
+            return this.count;
+        }
+
+        public double getCost() {
+            return this.cost;
+        }
+    }
+
+    /**
+     * Method, which fill massive by objects
+     */
+    public void setList() {

```

Comment:
This is not acceptable -to big and very messy. Must be splitted to methods/classes